parser/src/parser.rs

```diff
@@ -88,14 +88,17 @@ mod tests {
         assert_eq!(
             parse_ast,
             ast::Program {
-                statements: vec![ast::Statement::Expression {
-                    expression: ast::Expression::Call {
-                        function: Box::new(ast::Expression::Identifier {
-                            name: String::from("print"),
-                        }),
-                        args: vec![ast::Expression::String {
-                            value: String::from("Hello world"),
-                        },],
+                statements: vec![ast::Statement {
+                    location: ast::Location::new(1, 1),
```

Comment:
To avoid having to encode the indexing scheme here, `ast::Location` could implement the Default trait.

Reply:
What do you mean? In this test the location logic is actually tested, so having a default of 0, 0 would fail the test.